### PR TITLE
Add several spelling corrections

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -4602,6 +4602,10 @@ coelesce->coalesce
 coercable->coercible
 coerceion->coercion
 coersion->coercion
+coexhist->coexist
+coexhisted->coexisted
+coexhisting->coexisting
+coexhists->coexists
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Can you also separately add the hyphenated versions:
https://dictionary.cambridge.org/dictionary/english/co-exist

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -11133,6 +11139,7 @@ funchtion->function
 funchtional->functional
 funchtioned->functioned
 funchtioning->functioning
+funchtionning->functioning
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Can you add at least the -ed and possibly the -al and maybe -s variants separately too of the double n typo.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: Added in c1d0d79

Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: I also went ahead and did the same for some of the neighbor words as well.

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (luzpaz) [reply codespell_lib/data/dictionary.txt]: Added in c1d0d79

**Pull request merged**